Application/EdFi.Ods.AdminApp.Management/Helpers/ConnectionStrings.cs

```
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.AdminApp.Management.Helpers
+{
+    public class ConnectionStrings
+    {
+        public string Admin { get; set; }
+        public string Security { get; set; }
+        public string ProductionOds { get; set; }
+
+        public string GetConnectionStringByName(string databaseName)
+        {
+            switch (databaseName)
+            {
+                case CloudOdsDatabaseNames.Admin:
+                    return Admin;
+                case CloudOdsDatabaseNames.Security:
+                    return Security;
+                case CloudOdsDatabaseNames.ProductionOds:
+                    return ProductionOds;
+                default:
+                    return null;
```

This function is called by `GetConnectionStringFromConfigFile`, but I don't know what is calling `GetConnectionStringFromConfigFile`. Whatever that other thing is, what will it do if it gets a `null` here? I wonder if the default case should throw a runtime exception?

---

Refactored to throw an exception for an invalid database name.